src/material/select/select.spec.ts

```diff
@@ -1098,10 +1099,13 @@ describe('MatSelect', () => {
 
             fixture.componentInstance.control.setValue(fixture.componentInstance.foods[7].value);
             fixture.detectChanges();
-            trigger.click();
+            fixture.componentInstance.select.close();
             fixture.detectChanges();
             flush();
 
+            fixture.componentInstance.select.open();
+            fixture.detectChanges();
+
```

Comment:
The same changes will have to be made in the tests for the MDC-based select at `src/material-experimental/mdc-select/select.spec.ts`.